.github/dependabot.yml

```diff
@@ -10,6 +10,11 @@ updates:
 
   - package-ecosystem: "gomod"
     directory: "/"
+    groups:
+      aws-sdk-go:
+        patterns:
+          - "github.com/aws/aws-sdk-go"
```

Comment:
`github.com/aws/aws-sdk-go` isn't included in the root package

Reply:
Nice catch - removed this line.